apps/macos/package.json

```
@@ -32,11 +32,11 @@
     "jest": "^24.9.0",
     "metro-config": "^0.58.0",
     "metro-react-native-babel-preset": "^0.58.0",
-    "react-native": "^0.62.2",
+    "react-native": "0.63.3",
```

Should we use latest 0.63?
```suggestion
    "react-native": "0.63.4",
```